### ISSUE TITLE
install failed

### ISSUE THREAD

**Issue by saintthor** [PeculiarVentures/xadesjs#68]
```
/node-webcrypto-ossl$ sudo npm install
[!] 22 vulnerabilities found [637 packages audited]
    Severity: 15 Low | 5 Moderate | 1 High | 1 Critical



$ npm audit
 Critical      │ Command Injection     
 Package       │ growl                    
 Dependency of │ mocha [dev]  
 Path          │ mocha > growl       
 More info     │ https://nodesecurity.io/advisories/146  



$ sudo npm install mocha -g
402 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "mocha" "-g"
403 verbose node v10.0.0
404 verbose npm  v6.0.1
405 error code ERR_STREAM_WRITE_AFTER_END
406 error write after end
407 verbose exit [ 1, true ]
```

**Comment by microshine** [comment]
`mocha` is dev dependancy. It's not used in production version
Use `npm i node-webcrypto-ossl` instead  of `/node-webcrypto-ossl$ sudo npm install`

**Comment by microshine** [comment]
@saintthor I published new version `node-webcrypto-ossl` v1.0.37. It uses the lates version of `mocha`.

```
$ npm audit

                       === npm audit security report ===

[+] no known vulnerabilities found
    Packages audited: 2323 (2317 dev, 28 optional)```

**Comment by saintthor** [comment]
thank you. it works.
